scipy/optimize/_highs/setup.py

```diff
@@ -107,24 +115,25 @@ def configuration(parent_package='', top_path=None):
     # highs_wrapper:
     ipx_sources = _get_sources('src/CMakeLists.txt', 'set(ipx_sources\n', ')')
     highs_sources = _get_sources('src/CMakeLists.txt', 'set(sources\n', ')')
-    # filter out MIP sources until MIP is officially supported
-    highs_sources = [s for s in highs_sources
```

Comment:
The library seems to have dependencies on some MIP code, so I couldn't find a way to compile without these objects now